framework/vulkan_sample.h

```diff
@@ -253,21 +253,23 @@ class VulkanSample : public vkb::Application
 	 */
 	void create_render_context(const std::vector<SurfaceFormatType> &surface_priority_list);
 
-	DeviceType               &get_device();
-	DeviceType const         &get_device() const;
-	GuiType                  &get_gui();
-	GuiType const            &get_gui() const;
-	InstanceType             &get_instance();
-	InstanceType const       &get_instance() const;
-	RenderPipelineType       &get_render_pipeline();
-	RenderPipelineType const &get_render_pipeline() const;
-	sg::Scene                &get_scene();
-	StatsType                &get_stats();
-	SurfaceType               get_surface() const;
-	bool                      has_device() const;
-	bool                      has_gui() const;
-	bool                      has_render_pipeline() const;
-	bool                      has_scene();
+	DeviceType                           &get_device();
+	DeviceType const                     &get_device() const;
+	GuiType                              &get_gui();
+	GuiType const                        &get_gui() const;
+	std::vector<SurfaceFormatType>       &get_surface_priority_list();
+	std::vector<SurfaceFormatType> const &get_surface_priority_list() const;

```

Comment:
As there's no inherent order for all those functions, I sorted them alphabetically.